### PR TITLE
Add ibmjava

### PR DESCRIPTION
**Pull request by dinogun**
Add documentation support for ibmjava (IBM® SDK, Java™ Technology Edition) docker images. This pull request is specifically only for x86_64 arch.

It has 3 variants, the SDK, JRE and the SFJ (small footprint JRE). It currently uses ubuntu 16.04 and the JRE and the SFJ support a alpine variant as well.

### CODE REVIEW

**Comment by yosifkit** [comment]
LGTM

**Comment by dinogun** [comment]
Minor update to add the right URLs

File: ibmjava/content.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+##### Alpine Linux
+
+Consider using [Alpine Linux](http://alpinelinux.org/) if you are concerned about the size of the overall image. Alpine Linux is a stripped down version of Linux that is based on [musl glibc](http://wiki.musl-libc.org/wiki/Functional_differences_from_glibc) and Busybox, resulting in a [Docker image](https://hub.docker.com/_/alpine/) size of approximately 5 MB. Due to its extremely small size and reduced number of installed packages, it has a much smaller attack surface which improves security. However, because the IBM SDK has a dependency on gnu glibc, installing this library adds an extra 8 MB to the image size. The following table compares Docker Image sizes based on the latest JRE `8.0-3.10`.
```
Comment (tianon) [inline ibmjava/content.md]: `musl glibc` should be just `musl libc` (`glibc` is a specific `libc` implementation)

File: ibmjava/content.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Then start your JVM container by using `--volumes-from` flag to mount the shared volume, as shown in the following example:
+
+```console
+docker run -it --volumes-from classcache japp
+```
+
+### See Also
+
+See the [Websphere-Liberty image](https://hub.docker.com/_/websphere-liberty/), which builds on top of this IBM docker image for Java.
```
Comment (tianon) [inline ibmjava/content.md]: Woot, I was going to ask whether the plan was to update the `websphere-liberty` image to be `FROM ibmjava:...` :smile:

**Comment by tianon** [comment]
LGTM, just one minor issue that's easily fixed after the fact :+1:

**Comment by dinogun** [comment]
Thanks @tianon and @yosifkit ! Will fix the musc libc comment in the next iteration.

**Pull request merged**